docs/cody/clients/enable-cody-enterprise.mdx

```diff
@@ -1,17 +1,104 @@
+<p className="subtitle">This guide will walk you through the steps to install and set up Cody with your Sourcegraph Enterprise instance.</p>
 
-Cody enhances your coding experience by providing intelligent code suggestions, context-aware completions, and advanced code analysis. This guide will walk you through the steps to install and set up Cody with your Sourcegraph enterprise instance via:
-
-- [Self-hosted Sourcegraph](#cody-on-self-hosted-sourcegraph-enterprise)
-- [Sourcegraph Cloud](#cody-on-sourcegraph-cloud)
+Cody enhances your coding experience by providing intelligent code suggestions, context-aware completions, and advanced code analysis.
 
 <LinkCards>
    <LinkCard href="https://about.sourcegraph.com/cody/pricing" imgSrc="https://sourcegraph.com/.assets/img/sourcegraph-mark.svg" imgAlt="Cody Enterprise" title="Cody Enterprise" description="Get in touch with our team to try Cody for Sourcegraph Enterprise." />
 </LinkCards>
 
-## Cody on self-hosted Sourcegraph enterprise
+## Cody Enterprise features
+
+To cater to your Enterprise requirements, Cody Enterprise offers the following features:
+
+### IDE Token Expiry
+
+<Callout type= "info">IDE Token Expiry is supported with VS Code and JetBrains extensions.</Callout>
+
+Site administrators can customize the time duration of the access token used by other users on the Sourcegraph Enterprise instance to connect Cody from their IDEs via the **Site admin** page. Administrators can choose from various options, including 7, 14, 30, 60, and 90 days.
+
+![ide-token-expiry](https://storage.googleapis.com/sourcegraph-assets/Docs/token-expiry.png)
+
+### Guardrails
+
+<Callout type= "note">Guardrails for public code is currently in Beta stage and is supported with VS Code and JetBrains IDEs extensions.</Callout>
+
+Open source attribution guardrails for public code reduce the exposure to copyrighted code, commonly called copyright guardrails. This involves the implementation of a verification mechanism within Cody to ensure that any code generated by the platform does not replicate open source code.
+
+Guardrails for public code are available to all Sourcegraph Enterprise instances and are **disabled** by default. You can enable it from the **Site configuration** or the IDE extension settings. It only matches code snippets that are at least **10 lines** or longer, and it spans **290,000** search results.
+
+### Admin Controls
+
+<Callout type="note">Admin Controls is supported with VS Code IDE extension.</Callout>
+
+Sourcegraph account admins have selective control over users' access to Cody Enterprise, which is now managed via the Sourcegraph role-based access control system. This provides a more intuitive user interface for assigning permission to use Cody.
+
+Using the access controls, you can also configure which Cody capabilities users can have access to (for example, chat, completion, or commands).
```

Comment:
```suggestion

```
This only got implemented for VS Code so I don't think we should call it a capability